webapp/src/main/webapp/vue-apps/activity-stream/components/comment/list/ActivityCommentsDrawer.vue

```diff
@@ -268,6 +270,7 @@ export default {
         .then(data => {
           let comments = data && data.comments || [];
           this.commentsSize = data && data.size && Number(data.size) || 0;
+          this.totalCommentsSize = this.activity.totalCommentsSize = data?.totalCommentsSize && Number(data.totalCommentsSize) || 0;
```

Comment:
Seprate assignments into two different lines
```suggestion
          this.totalCommentsSize = data?.totalCommentsSize && Number(data.totalCommentsSize) || 0;
          this.activity.totalCommentsSize = this.totalCommentsSize;
```